test/integration/providers/docker.go

```diff
@@ -32,7 +33,7 @@ func MustRunDockerCommandAttach(container string, cmd string) {
 	}
 	defer cli.Close()
 
-	waiter, err := cli.ContainerAttach(ctx, container, types.ContainerAttachOptions{
+	waiter, _ := cli.ContainerAttach(ctx, container, types.ContainerAttachOptions{
```

Comment:
Should we check the `err` instead of ignoring it?

Reply:
done